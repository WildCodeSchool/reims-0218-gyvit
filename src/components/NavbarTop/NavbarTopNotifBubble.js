//TODO: create component notif bubble icon

import React from "react"
import PropTypes from "prop-types"
import { Badge } from "reactstrap"

const NavbarTopNotifBubble = () => {
  return (
    <Badge
      style={{
        marginTop: "-50px",
        backgroundColor: "#ff007e",
        width: "auto",
        height: "20px"
      }}
      pill
    >
      8
    </Badge>
  )
}

NavbarTopNotifBubble.propTypes = {
  notifsCount: PropTypes.number.isRequired
}

export default NavbarTopNotifBubble
