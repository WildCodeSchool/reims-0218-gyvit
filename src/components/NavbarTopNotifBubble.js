//TODO: create component notif bubble icon

import React from "react"
import { Badge } from "reactstrap"

const NavbarTopNotifBubble = ({ notifsCount }) => {
  return (
    <Badge color="danger" pill>
      {notifsCount}
    </Badge>
  )
}

export default NavbarTopNotifBubble
