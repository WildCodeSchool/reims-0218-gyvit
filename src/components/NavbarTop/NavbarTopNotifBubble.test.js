//TODO: Test component notif bubble icon

import React from "react"
import ReactDOM from "react-dom"
import renderer from "react-test-renderer"
import NavbarTopNotifBubble from "./NavbarTopNotifBubble"

const notifsCount = 6

it("renders without crashing", () => {
  const div = document.createElement("div")
  ReactDOM.render(<NavbarTopNotifBubble notifsCount={notifsCount} />, div)
  ReactDOM.unmountComponentAtNode(div)
})

it("renders correctly", () => {
  const tree = renderer
    .create(<NavbarTopNotifBubble notifsCount={notifsCount} />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
