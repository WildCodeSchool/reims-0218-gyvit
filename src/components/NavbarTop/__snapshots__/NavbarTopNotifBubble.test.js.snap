// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<span
  className="badge badge-secondary badge-pill"
  style={
    Object {
      "backgroundColor": "#ff007e",
      "height": "20px",
      "marginTop": "-50px",
      "width": "auto",
    }
  }
>
  8
</span>
`;
